src/app_core.py

```diff
@@ -158,7 +169,12 @@ def set_totp_secret(name, val):
 
 db = SQLAlchemy(app)
 migrate = Migrate(app, db)
-mail = MailSendGrid(app)
+#if os.getenv("USE_SENDGRID"):
+#if os.getenv("USE_SENDGRID") == 1:
+if os.getenv("USE_SENDGRID") and os.getenv("USE_SENDGRID") == "true":
```

Comment:
```suggestion
if app.config("USE_SENDGRID"):
```

Reply:
ahh there's a typo in USE_SENGRID, this should be USE_SENDGRID. making correction. and testing